docs/generate.html

```diff
@@ -667,7 +667,7 @@ <h2 class="mt-3">Contact Details</h2>
 
         <h2 class="mt-3">Information Sharing</h2>
         <p>
-          Defined as <em>sharing information with a legal entiry</em> typically
+          Defined as <em>sharing information with a legal entiyy</em> typically
```

Comment:
I was actually going for `entity`...

Reply:
Wow, I've managed to submit a one-character patch with an error in that character…